src/Parsing/String.purs

```diff
@@ -235,7 +238,7 @@ match p = do
 -- | capture the regular expression pattern `x*`.
 -- |
 -- | ```purescript
--- | case regex "x*" noFlags of
+-- | case regex "x*" multiline of
```

Comment:
Instead of changing the basic example, can you please instead write a new example in the **Flags** section which shows the effect of the `multiline` flag?